Catalog.html

```diff
@@ -3,12 +3,13 @@
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Седона</title>
+    <link rel="icon" href="favicon.ico">
   </head>
   <body>
     <header>
       <nav>
         <a href="index.html">
-          <img src="https://via.placeholder.com/138x70" width="138" height="70" alt="Логотип сайта городка Седона">
+          <img src="image\logo.png" width="138" height="70" alt="Логотип сайта городка Седона">
```

Comment:
```suggestion
          <img src="image/logo.svg" width="138" height="70" alt="Логотип сайта городка Седона">
```
1. В веб используется такой слеш.
2. Логотип нужно извлечь в формате SVG.